QUEST_JOBSTEP.tsv

```diff
@@ -753,42 +753,42 @@ QUEST_JOBSTEP_20150317_000752	The disciples came to Lydia Schaffen. Afterwards,
+$QUEST_JOBSTEP_20150317_000762	One day Lydia Schaffen sneaked out of a certain village alone and walked around at night.{nl} Lydia Schaffen saw a traveler sitting on a woodland road. {nl} Out of curiosity, she decided to ask him.{nl} 'Why are you sitting here in the forest in the middle of the night?'{nl}The traveler was hesitant to answer up to this point, but after being asked multiple times he finally opened his mouth. {nl}The reason was apparently this. {nl}
+$QUEST_JOBSTEP_20150317_000763	The traveler had met a passing demon, named Rumpelstiltskin, in the forest several days ago. He was invited to play a game with the demon. 
+$QUEST_JOBSTEP_20150317_000764	The reward for winning the game was significant, but the game with the demon was not easy. In the end, the traveler lost to Rumpelstiltskin. And when traveler reached the part where he had to pay for losing the game, he took a short breath, he was told to keep waiting. After hearing the conditions, Lydia Schaffen started to search for the troublemaker, Rumpelstiltskin.
+$QUEST_JOBSTEP_20150317_000765	Lydia quickly liberated the traveler of his conditions, and in exchange, sent a challenge request to Rumpelstiltskin. Rumpelstiltskin was well acquainted with Lydia Schaffen's reputation, and the fact that she requested a game with him excited him into accepting her request. 
+$QUEST_JOBSTEP_20150317_000766	Although Rumpelstiltskin is not well known by people today, he played often and was famous for gifting those who lost in his games in the past. But, his games have always ended with little variation. Lydia Schaffen proposed ,"I ask for a game related to archery, but, what sort of thing should we do?"
+$QUEST_JOBSTEP_20150317_000767	"Of course, a game to see who can shoot better with a bow would not be that interesting. Therefore, let's play a game where you fire an arrow with your bow at the target, and then, I will try to catch and ride on the arrow before the arrow reaches the target," said Rumpelstiltskin. Lydia replied back, questioning the proposal,"Ride on the arrow?" "That is correct," averred Rumpelstiltskin.
+$QUEST_JOBSTEP_20150317_000768	Upon answering, Rumpelstiltskin started to shrink his body. Once he had reduced himself to the size of a child's fist, Rumpelstiltskin spoke ,"Well, this size will allow me to ride on your arrow, right?" Lydia Schaffen thought for a bit ,"That's a problem for the match. If you touch the arrow and cause it to fall to the ground without it ever hitting the target, then wouldn't that still be a loss for me regardless of whether you rode the arrow or not?" Lydia answered Rumpelstiltskin with these words.
+$QUEST_JOBSTEP_20150317_000769	"Then how about this. I win if I can ride your arrow until it reaches the target," answered Rumpelstiltskin. "And if your arrow falls to the ground," he continued ,"then I will also lose." Rumpelstiltskin added ,"The game shall have 3 rounds, best 2 out of 3."
```

Comment:
he continued ,"then I will also lose." Rumpelstiltskin added ,"The game shall have 3 rounds, best 2 out of 3." -> he continued, "then I will also lose." Rumpelstiltskin added, "The game shall have 3 rounds, best 2 out of 3."